### PR TITLE
Spotlight checkboxes for people who've read a post

### PR DESCRIPTION
**Pull request by Raemon**
If a user has read any posts in a sequence, they see checkboxes showing which ones they've read:

┆Issue is synchronized with this [Asana task](https://app.asana.com/0/1201302964208280/1203069480577517) by [Unito](https://www.unito.io)

### CODE REVIEW

File: packages/lesswrong/components/sequences/BooksProgressBar.tsx [hunk 1]
```diff
@@ -6,17 +6,21 @@ import classNames from 'classnames';
 import { useItemsRead } from '../common/withRecordPostView';
 import { forumTypeSetting } from '../../lib/instanceSettings';
 
+export const postProgressBoxStyles = theme => ({
```
Comment (b0b3rt) [inline packages/lesswrong/components/sequences/BooksProgressBar.tsx:9]: `theme` param should be typed

File: packages/lesswrong/components/spotlights/SpotlightItem.tsx [hunk 2]
```diff
@@ -4,13 +4,18 @@ import EditIcon from '@material-ui/icons/Edit';
 import MoreVertIcon from '@material-ui/icons/MoreVert';
 import classNames from 'classnames';
 import React, { useState } from 'react';
+import { post } from 'request';
```
Comment (b0b3rt) [inline packages/lesswrong/components/spotlights/SpotlightItem.tsx:7]: unused, I think (hope)?

File: packages/lesswrong/components/spotlights/SpotlightItem.tsx [hunk 3]
```diff
@@ -4,13 +4,18 @@ import EditIcon from '@material-ui/icons/Edit';
...
 import Spotlights from '../../lib/collections/spotlights/collection';
+import { useMulti } from '../../lib/crud/withMulti';
+import { forumTypeSetting } from '../../lib/instanceSettings';
 import { Link } from '../../lib/reactRouterWrapper';
 import { Components, getFragment, registerComponent } from '../../lib/vulcan-lib';
 import { userCanDo } from '../../lib/vulcan-users';
 import { postBodyStyles } from '../../themes/stylePiping';
+import { useItemsRead } from '../common/withRecordPostView';
 import { useCurrentUser } from '../common/withUser';
+import { postProgressBoxStyles, postProgressRead } from '../sequences/BooksProgressBar';
```
Comment (b0b3rt) [inline packages/lesswrong/components/spotlights/SpotlightItem.tsx:18]: `postProgressRead` isn't used

File: packages/lesswrong/components/spotlights/SpotlightStartOrContinueReading.tsx [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+    const { LWTooltip, PostsPreviewTooltip} = Components
+  const { results: chapters } = useMulti({
+    terms: {
+      view: "SequenceChapters",
+      sequenceId: spotlight.documentId,
+      limit: 100
+    },
+    collectionName: "Chapters",
+    fragmentName: 'ChaptersFragment',
+    enableTotal: false,
```
Comment (Raemon) [inline packages/lesswrong/components/spotlights/SpotlightStartOrContinueReading.tsx:53]:
```
```

Comment (Raemon) [inline packages/lesswrong/components/spotlights/SpotlightStartOrContinueReading.tsx:53]:
```suggestion
   
```

**Review by b0b3rt — APPROVED** [review]

**Pull request merged**